lib/mosaic-utils/src/main/java/org/eclipse/mosaic/lib/util/ProcessLoggingThread.java

```diff
@@ -22,23 +22,19 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
+import java.util.function.Consumer;
 
 public class ProcessLoggingThread extends Thread {
 
-    public enum Level { Error, Info, Debug, Trace }
-
+    private final String processName;
+    private final Consumer<String> lineConsumer;
     private final InputStream stream;
-    private final org.slf4j.Logger log;
-    private final String id;
-    private final Level level;
     private boolean running = true;
 
-    public ProcessLoggingThread(org.slf4j.Logger log, InputStream stream, String identifier, Level level) {
+    public ProcessLoggingThread(String processName, InputStream stream, Consumer<String> lineConsumer) {
```

Comment:
Did the `identifier` ever have a use?

Reply:
This is now renamed to "processName" and is indeed useful to have the processes name in the log statement.

Reply:
Oh yeah, I thought the `processName` parameter was there before the change. My bad!